src/main/webapp/services/commentService.js

```diff
@@ -93,10 +93,6 @@ export default class CommentService extends BaseService {
       this.updateDialog('commentIssue', template.layout, template.data);
     } else {
       dialogBuilder.disableButtons(true);
-
-      const template = this.retrieveTemplate(dialogBuilder, data, this.serviceName, 'SAVING...');
-      this.updateDialog('commentIssue', template.layout, template.data);
```

Comment:
If you remove those statements probably you won't update the dialog and the buttons no longer blocked until finish the assignment process. Perhaps, you should set the comment value before update the dialog.